### PR TITLE
Added Hindi

### PR DESCRIPTION
**Pull request by AshokChakra-TJ**
Added Hindi. Please review.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -782,6 +782,26 @@
...
+        lang_dir = 'ltr',
+        database = 'hiwiki_p',
+        wikipedia_domain = 'hi.wikipedia.org',
+
+        beginners_link = 'https://hi.wikipedia.org/wiki/सहायता:यथादृश्य_संपादिका_के_साथ_संदर्भ_देने_के_लिए_परिचय/१',
+        beginners_link_title = 'यथादृश्य संपादिका के साथ संदर्भ देने के लिए परिचय',
+        lead_section_policy_link = '',
+        lead_section_policy_link_title = '',
+
+        citation_needed_category = 'सभी_लेख_जिनमें_स्रोतहीन_कथन_हैं',
```
Comment (eggpi) [inline config.py]: This category seems to be empty? What is the difference between this one and स्रोतहीन_कथनों_वाले_सभी_लेख, which was suggested in #131 and seems to actually contain articles?

File: config.py [hunk 2]
```diff
@@ -782,6 +782,26 @@
...
+        citation_needed_templates = [
+            'उद्धरण आवश्यक',
+            'Citation needed',
+            'cn'
```
Comment (eggpi) [inline config.py]: No need to specify "Citation needed" and "cn" -- they're common to all wikis so we already handle them in all languages :)

File: config.py (hunk 1 above)
Comment (AshokChakra-TJ) [reply config.py]: Hey see [this](https://hi.wikipedia.org/wiki/%E0%A4%B6%E0%A5%8D%E0%A4%B0%E0%A5%87%E0%A4%A3%E0%A5%80:%E0%A4%B8%E0%A4%AD%E0%A5%80_%E0%A4%B2%E0%A5%87%E0%A4%96_%E0%A4%9C%E0%A4%BF%E0%A4%A8%E0%A4%AE%E0%A5%87%E0%A4%82_%E0%A4%B8%E0%A5%8D%E0%A4%B0%E0%A5%8B%E0%A4%A4%E0%A4%B9%E0%A5%80%E0%A4%A8_%E0%A4%95%E0%A4%A5%E0%A4%A8_%E0%A4%B9%E0%A5%88%E0%A4%82) (सभी_लेख_जिनमें_स्रोतहीन_कथन_हैं) it contains 60 subcategories and many articles. And the category mentioned in #131 is for {{Failed verification}}. I hope now its clear. Thanks for reviewing! :)

File: config.py (hunk 2 above)
Comment (AshokChakra-TJ) [reply config.py]: Oh, that's cool! Change it according to you.

File: config.py (hunk 1 above)
Comment (eggpi) [reply config.py]: Oh, my bad. I was trying to double-check the category but made some mistake when copy-pasting the name before. Yes, I can see that this is right :)

**Comment by eggpi** [comment]
Thanks, merging now! If all is well, this should show up in a few minutes.

**Pull request merged**